app/config/Config.scala

```diff
@@ -1,37 +1,43 @@
 package config
 
-import com.amazonaws.auth.{AWSCredentialsProviderChain, InstanceProfileCredentialsProvider}
 import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.auth.{AWSCredentialsProviderChain, InstanceProfileCredentialsProvider}
+import com.amazonaws.regions.Region
 import play.api.Configuration
 import services.AwsInstanceTags
 
 class Config(config: Configuration) extends AwsInstanceTags {
 
-  val stage = readTag("Stage") getOrElse "DEV"
-  val appName = readTag("App") getOrElse "editorial-production-metrics"
-  val stack = readTag("Stack") getOrElse "flexible"
-  val region = services.EC2Client.region
+  val stage: String = readTag("Stage") getOrElse "DEV"
+  val appName: String = readTag("App") getOrElse "editorial-production-metrics"
+  val stack: String = readTag("Stack") getOrElse "flexible"
+  val region: Region = services.EC2Client.region
 
   val awsCredentialsProvider = new AWSCredentialsProviderChain(
     new ProfileCredentialsProvider("composer"),
     new InstanceProfileCredentialsProvider(false)
   )
+  val elkKinesisStream: String = getConfigString("elk.kinesis.stream")
+  val elkLoggingEnabled: Boolean = getConfigBoolean("elk.logging.enabled", Some(true))
 
-  def getConfigString(key: String): String = config.getString(key)
-    .getOrElse(throw new RuntimeException(s"key $key not found in configuration"))
+  val pandaDomain: String = getConfigString("panda.domain")
+  val pandaAuthCallback: String = getConfigString("panda.authCallback")
+  val pandaSystem: String = getConfigString("panda.system")
 
-  val elkKinesisStream = getConfigString("elk.kinesis.stream")
-  val elkLoggingEnabled = config.getBoolean("elk.logging.enabled").getOrElse(true)
+  val publishingMetricsKinesisStream: String = getConfigString("kinesis.publishingMetricsStream")
 
-  val pandaDomain = getConfigString("panda.domain")
-  val pandaAuthCallback = getConfigString("panda.authCallback")
-  val pandaSystem = getConfigString("panda.system")
+  val tagManagerUrl = s"${getConfigString("tagManager.url", Some("http://tagmanager.gutools.co.uk"))}/hyper/tags"
 
-  val publishingMetricsKinesisStream = getConfigString("kinesis.publishingMetricsStream")
+//  This is for uniquely identifying the kinesis application when running the app locally on multiple DEV machines
+  val devIdentifier: String = if(stage == "DEV") getConfigString("user") else ""
 
-  val tagManagerUrl = s"${config.getString("tagManager.url").getOrElse("http://tagmanager.gutools.co.uk")}/hyper/tags"
+  val workflowUrl: String =  getConfigString("workflow.url")
 
-//  This is for uniquely identifying the kinesis application when running the app locally on multiple DEV machines
-  val devIdentifier = if(stage == "DEV") getConfigString("user") else ""
+  val hmacSecret: String = getConfigString("hmacSecret")
```

Comment:
The hmac secret will need to go in dynamo and in the 2 config files s3 for local development (I always forget one of them)

Reply:
I have a list of all the new config values we need to add, I'll make sure I update everything before merging.